File: 1. Escuela Desarrollo Web/25-curso-POO/CursoPOOUber/Python/main.py

from car import Car
from account import Account

if __name__ == "__main__":
    print("Hola Mundo")
    
    car = Car("AMS234", Account("Andres Herrera", "ANDA876"))
    print(vars(car))
    print(vars(car.driver))

    # car2 = Car()
    # car2.id = 2
    # car2.license = "BBB123"
    # car2.driver = "Muchisx"
    # car2.passengers = 4
    # print(vars(car2))

File: 1. Escuela Desarrollo Web/25-curso-POO/CursoPOOUber/Java/Main.java
class Main {

    public static void main(String[] args) {
        System.out.println("Hey");
        Car car = new Car("AAA123", new Account("Muchis", "CC1"));
        car.id = 1;
        car.passengers = 4;
        car.printDataCar();

        Car car2 = new Car("BBB123", new Account("Muchisx", "CC2"));
        car2.id = 2;
        car2.passengers = 4;
        car2.printDataCar();
    }
}

// Name of the class will be set to the same of the file
